packages/docHandler/package.json

```diff
@@ -1,48 +1,49 @@
-    "start:dapr:test": "dapr run --app-id docHandler --app-protocol http --app-port 3001 --dapr-http-port 3501 --config ./dapr-config.yaml --placement-host-address localhost:50006 --resources-path ../../daprComponents/dev node test",
-    "start:dapr:dev": "dapr run --app-id docHandler --app-protocol http --app-port 3001 --dapr-http-port 3501 --config ./dapr-config.yaml --placement-host-address localhost:50006 --resources-path ../../daprComponents/dev yarn start:dev"
-  },
-  "dependencies": {
-    "@a-la-fois/models": "0.0.29",
-    "@dapr/dapr": "3.0.0",
-    "@typegoose/typegoose": "11.2.0",
-    "js-base64": "3.7.5",
-    "mongoose": "7.2.2",
-    "yjs": "13.6.1"
-  },
-  "devDependencies": {
-    "nodemon": "2.0.22",
-    "rimraf": "5.0.1",
-    "ts-node": "10.9.1",
-    "typescript": "5.1.3"
-  },
-  "gitHead": "e757a752497e9c4ee4ad85603998907ef3d54c22"
+    "name": "@a-la-fois/doc-handler",
+    "version": "0.0.29",
+    "license": "MIT",
+    "files": [
+        "lib"
+    ],
+    "main": "./lib/cjs/library/index.js",
+    "module": "./lib/esm/library/index.js",
+    "types": "./lib/esm/library/index.d.ts",
+    "exports": {
+        "require": "./lib/cjs/library/index.js",
+        "import": "./lib/esm/library/index.js",
+        "types": "./lib/esm/library/index.d.ts"
+    },
+    "scripts": {
+        "build": "yarn build:lib && yarn build:server",
+        "pub": "yarn publish",
+        "build:server": "tsc --build",
+        "build:lib": "tsc --project ./tsconfig.build.lib.cjs.json && tsc --project ./tsconfig.build.lib.esm.json",
+        "build:lib:dev": "tsc --project ./tsconfig.build.lib.cjs.json --watch --preserveWatchOutput & tsc --project ./tsconfig.build.lib.esm.json --watch --preserveWatchOutput",
+        "prebuild:server": "rimraf dist",
+        "prebuild:lib": "rimraf lib",
+        "publish:image": "./scripts/dockerPublish.sh",
+        "prepare": "npm run build:lib",
+        "start": "node dist/main.js",
+        "start:dev": "nodemon --ext \".ts,.js\" --watch \"./src\" --exec \"ts-node ./src/main.ts\" & yarn build:lib:dev",
+        "clear": "rimraf dist lib",
+        "start:dapr": "dapr run --app-id docHandler --app-protocol http --app-port 3001 --dapr-http-port 3501 --config ./dapr-config.yaml --placement-host-address localhost:50006 --resources-path ../../daprComponents/dev yarn start",
+        "start:dapr:test": "dapr run --app-id docHandler --app-protocol http --app-port 3001 --dapr-http-port 3501 --config ./dapr-config.yaml --placement-host-address localhost:50006 --resources-path ../../daprComponents/dev node test",
+        "start:dapr:dev": "dapr run --app-id docHandler --app-protocol http --app-port 3001 --dapr-http-port 3501 --config ./dapr-config.yaml --placement-host-address localhost:50006 --resources-path ../../daprComponents/dev yarn start:dev"
+    },
+    "dependencies": {
```

Comment:
in json must be 2 spaces tab
it's in prettier config and npm uses 2 spaces. If to change to 4 npm will return it to 2 after next manipulation with package.json

Reply:
Yeah, I guess I need to change my editor settings. I'll come back with a patch